projects/power_distribution/inc/smoketests_pd.h

```diff
@@ -8,6 +8,7 @@
 #define PD_SMOKE_TEST SMOKE_TEST_CURRENT_MEASUREMENT
```

Comment:
Oops, looks like I accidentally left this uncommented. Could you comment this out so that by default power distribution works normally rather than going into the current measurement smoke test?